source/features/repo-icon-buttons.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import PlusIcon from 'octicon/plus.svg';
+import SearchIcon from 'octicon/search.svg';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {groupButtons} from '../github-helpers/group-buttons';
+
+function init(): void {
+	const searchButton = select('.file-navigation a.btn.d-md-block')!;
```

Comment:
All of these selectors are too generic, you’ll have to select something more specific. If necessary, select a child then use `closest` to get what you want.